beacon-chain/execution/engine_client.go

```diff
@@ -701,24 +735,51 @@ func fullPayloadFromExecutionBlock(
 			BlockHash:     blockHash[:],
 			Transactions:  txs,
 		})
+	case version.Capella:
```

Comment:
There don't appear to be any tests covering the Capella and Deneb cases

Reply:
c8cf56fb3